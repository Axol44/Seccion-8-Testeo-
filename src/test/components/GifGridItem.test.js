import React from 'react';
import { shallow } from 'enzyme'
import { GifGridItem } from '../../components/GifGridItem';


describe('Pruebas en <GifGridItem />', () => {

  const title = 'Un titulo';
  const url = 'https://localhost/otracosa.jpg'
  const wrapper = shallow( < GifGridItem title={ title } url={ url } /> );

  test('Debe mostral el componente correctamnete', () => { 
    expect( wrapper ).toMatchSnapshot();
   })
  
  test('Debe tener un pÃ¡rrafo con el title ', () => { 
    const p = wrapper.find('p');
    expect( p.text().trim() ).toBe( title );
  })   
   
} )