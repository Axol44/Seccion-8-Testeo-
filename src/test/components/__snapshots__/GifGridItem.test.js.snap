// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> Debe mostral el componente correctamnete 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Un titulo"
    src="https://localhost/otracosa.jpg"
  />
  <p>
     
    Un titulo
     
  </p>
</div>
`;
